src/progressive-web-app.ts

```diff
@@ -1,15 +1,55 @@
-self.addEventListener("install", (event: InstallEvent) => {
+const CACHE_NAME = "devpool-directory-cache-v1";
```

Comment:
Maybe it's better to export this and import where needed to increase maintainability.

Reply:
Should I create a constant file for this or use the already github-types file for it?

Reply:
Either I think is fine.